src/tasks/register-authorizers.js

```diff
@@ -78,7 +79,7 @@ module.exports = function registerAuthorizers(authorizerMap, apiId, awsRegion, f
 			if (authConfig.credentials) {
 				params.authorizerCredentials = authConfig.credentials;
 			}
-			if (authConfig.resultTtl) {
+			if (authConfig.resultTtl !== undefined) {
```

Comment:
maybe better to use Number.isInteger(authConfig.resultTtl), that's true for 0 and numbers, this way a string or something else could come through